fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/serializer/JsonCastVoteDeserializer.java

```diff
@@ -38,12 +37,13 @@ public CastVote deserialize(JsonElement json, Type typeOfT, JsonDeserializationC
         boolean typeValidationString = true;
         // Vote type of a CastVote is either an integer for an OpenBallot election or a
         // String for an Encrypted election, type should be valid for all votes
+        String voteIndex = "vote";
```

Comment:
It should be a public static final String of the class

Reply:
ohhh I see, why not private static, I think we won't use it anywhere else

Reply:
Then private is fine